### DISCUSSION TITLE
How do I deploy without building?

### DISCUSSION THREAD

**Discussion by ericelliott** [vercel/community#3811] [Help] [answered]
### Summary

I have a project that gets pre-built into a `dist` folder before I try to deploy with vercel. I want to skip vercel's build step entirely. I have tried:

![Screen Shot 2023-08-28 at 10 18 22 AM](https://github.com/vercel/community/assets/364727/b8c95261-5637-4c48-a49a-ab93ce8e67b0)

And:

vercel.json
```
{
  "buildCommand": "echo 'skipping build with vercel.json'"
}
```

I have both in my project, but it still tries to build when I run `vercel` to deploy:

<img width="1788" alt="Screen Shot 2023-08-28 at 10 21 07 AM" src="https://github.com/vercel/community/assets/364727/41df50d8-c5d3-4b44-9d4e-b1f4e5548e37">

I do not need it to try to run `yarn install` during deploy. My project is pre-built.

Please help. :please

### Example

https://emojimon-client-4tpr729ow-ericelliott.vercel.app/_logs

### Steps to Reproduce

see above

**Comment by amyegan** [comment, marked as answer]
Hey @ericelliott. There are a few options. The right one for you will depend on your site and what makes most sense to you. I can think of a couple of options that may work here.

Since you have a local build step and just want to deploy the prebuilt output, you could use the CLI: https://vercel.com/docs/cli/deploying-from-cli#deploying-from-local-build-prebuilt

If you push the `dist` folder to your remote repo for automatic deployment, then you could override the root directory build command in the project's [build and development settings](https://vercel.com/docs/deployments/configure-a-build#build-and-development-settings). You can enable the "override" option and leave the command empty to [skip the build step](https://vercel.com/docs/deployments/configure-a-build#skip-build-step) and serve content directly.

In either case, you should disable the Ignored Build Step option since it will prevent the project from being deployed.
